WalletWasabi.Fluent/ViewModels/Wallets/Home/Tiles/PrivacyRing/PrivacyRingViewModel.cs

```diff
@@ -74,82 +72,61 @@ protected override void OnNavigatedTo(bool isInHistory, CompositeDisposable disp
 						.PrivacyProgressUpdateTrigger
 						.CombineLatest(sizeTrigger)
 						.ObserveOn(RxApp.MainThreadScheduler)
-						.Subscribe(x =>
-						{
-							var usableHeight = x.Second.Item2;
-							var usableWidth = x.Second.Item1;
-							Margin = new Thickness(usableWidth / 2, usableHeight / 2, 0, 0);
-							NegativeMargin = new Thickness(Margin.Left * -1, Margin.Top * -1, 0, 0);
-							RefreshCoinsList(itemsSourceList);
-						})
+						.Subscribe(_ => RenderRing(itemsSourceList))
```

Comment:
Nit, but since the passed values are not used anymore, you can convert the `sizeTrigger` with `ToSignal()` and replace the `CombineLatest` with a simple `Merge`.

Reply:
And I would also add 
```c#
.Do(_ => itemsSourceList.Clear())
```
line to the `sizeTrigger`, which would fix the weird layout glitch when you resize the window.
The decision is yours if you include it in this PR or in a separate one.